### PR TITLE
add State Maintenance, mostly handwaving text.

### PR DESCRIPTION
**Pull request by danwing**

### CODE REVIEW

**Review by tireddy2 — APPROVED** [review]

File: draft-wing-cidfi.md [hunk 1]
```diff
@@ -897,6 +897,17 @@ network attach and which DCIDs are for a migrated connection.  Probably
 belongs in the QUIC transport parameter signaling?
 
 
+# State Maintenance
+
+The CNE can safely remove state after UDP inactivity timeout {{Section
```
Comment (boucadair) [inline draft-wing-cidfi.md:902]:
```suggestion
A CNE can safely remove state after UDP inactivity timeout ({{Section
```

File: draft-wing-cidfi.md [hunk 2]
```diff
@@ -897,6 +897,17 @@ network attach and which DCIDs are for a migrated connection.  Probably
...
+4.3 of !RFC4787}}.  The CIDFI client MUST re-signal its CNE when it
```
Comment (boucadair) [inline draft-wing-cidfi.md:903]:
```suggestion
4.3 of !RFC4787}}).  The CIDFI client MUST re-signal its CNE(s) when it
```

File: draft-wing-cidfi.md [hunk 3]
```diff
@@ -897,6 +897,17 @@ network attach and which DCIDs are for a migrated connection.  Probably
...
+receives a QUIC path validation message, as that indicates a NAT
+rebinding occurred.  CNE state can also be cleared by signaling from
```
Comment (boucadair) [inline draft-wing-cidfi.md:905]:
```suggestion
rebinding occurred.  A CNE's state can also be cleared by signaling from
```

**Review by boucadair — COMMENTED** [review]
Minor edits as many CNEs may be involved.

**Pull request merged**